lib/git.js

```diff
@@ -60,13 +60,21 @@ exports.getCommits = function (options) {
 
     var parsed = commit.subject.match(COMMIT_PATTERN);
 
-    if (!parsed || !parsed[1] || !parsed[4]) {
+    if (!parsed || !parsed[1] || !parsed[3]) {
       return null;
     }
 
     commit.type = parsed[1].toLowerCase();
-    commit.category = parsed[3] || '';
-    commit.subject = parsed[4];
+    commit.category = parsed[2] || '';
+    commit.subject = parsed[3];
+
+    if (parsed[4]) {
+      parsed[4].toLowerCase().split(/\s*,\s*/).forEach(function (flag) {
+        switch (flag) {
+          case 'breaking': commit.type = flag;
```

Comment:
I know it doesn't matter here too much since it's just one case and it's a one-liner, but can you format it like so:

```js
switch (flag) {
  case 'breaking':
    commit.type = flag;
    break;
}
```

Reply:
👍